### DISCUSSION TITLE
Fix common HASPone issues with Home Assistant

### DISCUSSION THREAD

**Discussion by aderusha** [HASwitchPlate/HASPone#104] [General]
I've noticed two common issues that seem to be cropping up a lot and which result in failed automations in Home Assistant.  I've created [a new wiki page here](https://github.com/HASwitchPlate/HASPone/wiki/Fix-common-HASPone-issues) with some steps you can take to help resolve these issues.

**Comment by RCPilot1604** [comment]
Wow! This post really needs an upvote. Searched high and low on various forums until I found this which immediately fixed the issue! Thanks a lot!

**Reply by aderusha** [reply]
Hey glad it helped :D  Did this resolve the problem you encountered [here](https://github.com/HASwitchPlate/HASPone/discussions/133) or do you still need some help on that?

**Reply by RCPilot1604** [reply]
It is resolved. Thanks!
